cpp/daal/src/externals/core_threading_win_dll.cpp

```diff
@@ -277,8 +277,8 @@ typedef void (*_daal_tbb_task_scheduler_free_t)(void *& globalControl);
 typedef size_t (*_setNumberOfThreads_t)(const size_t, void **);
 typedef void * (*_daal_threader_env_t)();
 
-typedef void (*_daal_parallel_sort_int_t)(int *, int *);
-typedef void (*_daal_parallel_sort_size_t_t)(size_t *, size_t *);
+typedef void (*_daal_parallel_sort_int32_t)(std::int32_t *, std::int32_t *);
+typedef void (*_daal_parallel_sort_uint64_t)(std::uint64_t *, std::uint64_t *);
```

Comment:
In fact we prefer to use int and size_t in DAAL.

Reply:
fixed